scripts/deletion.py

```diff
@@ -177,7 +187,12 @@ def _remove_object_from_gs(client, indexclient, f, target_bucket):
         list(DeletionLog)
 
     """
+    object_key = get_structured_object_key(f, IGNORED_FILES)
```

Comment:
should this be `utils.get_ignored_files(IGNORED_FILES)` instead of `IGNORED_FILES`?

Reply:
Good catch! Thanks
fa197bd2c7f245e7a0fc3771be8044f233b9656d